.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run lint-staged
+# npm run lint-staged
```

Comment:
😭

Reply:
Savage

Reply:
My POV, but git CLI hooks are a pain for outside contributors on public projects.

Someone fixing a bug/adding a feature should just have to worry about fixing the bug/adding the feature. Then the lint CI would eventually fail on the PR and maintainers can help to straighten that before merging because they're the ones who are familiar with their linting rules and know how to fix that quickly~